### DISCUSSION TITLE
.io 一個越來越紅的網域名稱

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#623] [科技新知]
https://lab.howie.tw/2014/01/why-io-domain-is-hot.html
當初發現這個的原因是在玩 gratic.io的時候，漸漸地了解到許多NFT的介紹頁面也都是.io，於是查一查，才查到io的i指的是input，o者的是output，由於比.com更少字，所以泛用性也逐漸提高。

**Comment by HsuChen88** [comment]
因為.com的網域較貴且日漸飽和了

**Comment by ghost** [comment]
原來還有這樣的區分 長姿勢了

**Comment by henrywen547** [comment]
我最一開始接觸到.io的網域也是在一些slither.io等遊戲上，我當時還真沒有思考.io的意義，原來是input/output啊!

**Comment by Rosancl** [comment]
長知識惹 玩過好幾個.io的遊戲，現在才知道.io的意思~~

**Comment by gnsJhenJie** [comment]
大家如果有 GitHub Student Developer Pack 的話，可以免費拿到一年的.me和.tech 網域
https://education.github.com/pack?sort=popularity&tag=Domains

**Comment by bloodtrail0511** [comment]
推幾個io遊戲:[動物大作戰](mope.io)、[diep.io](diep.io)、[krunker.io](krunker.io)

**Comment by jason79461385** [comment]
自己有用過draw.io畫流程圖....大一時候的惡夢。

**Comment by yucheng1028** [comment]
原來如此 小時候電腦課都在玩.io的遊戲
